### ISSUE TITLE
Studio: resize shape reference window?

### ISSUE THREAD

**Issue by omgitsraven** [libfive/libfive#208]
I just launched Studio for the first time in a while, and it seems like it isn't possible to resize the Shape Reference window any more? I'm sure I used to be able to make it taller, to glance through more entries at once...

**Comment by sjm-tl-gh** [comment]
Resizing of the shape reference window is still working for me on the following system:

Ubuntu 18.04
Libfive Revision: 01ace4b (latest commit as of today)
Qt package: 5.9.5 (latest Ubuntu 18.04 version)

**Comment by omgitsraven** [comment]
Still not working for me on

macOS 10.13.6
01ace4b
Qt 5.11.1

**Comment by omgitsraven** [comment]
Also: I reverted back to Qt 5.9.3 and rebuilt Studio, and the bug is still happening, so that's not it.

**Comment by mkeeter** [comment]
Yup, I'm seeing it as well @omgitsraven – I'll see what I can find.
